src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/utils/TransactionUtils.java

```diff
@@ -17,6 +18,18 @@ public boolean isTransactionLinkedToLimitedPartnershipSubmission(Transaction tra
         return doChecks(transaction, limitedPartnershipSubmissionSelfLink, FILING_KIND_LIMITED_PARTNERSHIP);
     }
 
+    public boolean doesTransactionHaveALimitedPartnershipSubmission(Transaction transaction) {
```

Comment:
I think we need to move away from the name 'Submission' so it matches the architecture design

Reply:
Yes, but same applies here - method above should also change its name (`isTransactionLinkedToLimitedPartnershipSubmission`) which will then impact on other places in the codebase.

Shall I create a new API refactoring story to pick all this improvement work up. It could also include the `TransactionUtils` refactoring mentioned by @lduranteau above?

Reply:
yes probably better to do in a separate task